src/controllers/identity-hub-controllers/participants-controller.ts

```diff
@@ -0,0 +1,62 @@
+import { EdcConnectorClientContext } from "../../context";
+import { Participant, ParticipantInput } from "../../entities/identity-hub";
+import { Inner } from "../../inner";
+import { ParticipantController } from "./participant-controller";
+
+export class ParticipantsController {
+  #inner: Inner;
+  #context?: EdcConnectorClientContext;
+
+  constructor(inner: Inner, context?: EdcConnectorClientContext) {
+    this.#inner = inner;
+    this.#context = context;
+  }
+
+  async queryAll(
+    query: { offset?: string; limit?: string } = {},
+    context?: EdcConnectorClientContext,
+  ) {
+    const actualContext = context || this.#context!;
+
+    return this.#inner.request<Participant[]>(actualContext.identity, {
+      path: "/v1alpha/participants",
+      method: "GET",
+      apiToken: actualContext.apiToken,
+      query,
+    });
+  }
+
+  async create(input: ParticipantInput, context?: EdcConnectorClientContext) {
+    const actualContext = context || this.#context!;
+
+    return this.#inner.request<{
+      apiKey: string;
+      clientId: string;
+      clientSecret: string;
+    }>(actualContext.identity, {
+      path: "/v1alpha/participants",
+      method: "POST",
+      apiToken: actualContext.apiToken,
+      body: input,
+    });
+  }
+
+  async get(participantId: number, context?: EdcConnectorClientContext) {
```

Comment:
I think the "participant data" and the "participant controller" should be obtained with separated methods.
like:
- `get(participantId)`: do the HTTP call and return the Participant data object
- `participant(participantId)`: returns the controller without doing any call

this because there's should need to fetch a participant to access its sub-resources as dids, keys and so on, plus somebody would need to get the Participant data, and now it's not possible